examples/http-caching-client/http_caching_client.server.out

```diff
@@ -79,7 +79,7 @@ host: localhost:8080
 connection: keep-alive
 content-length: 0
 
-# The response has not changed. Therefore the backend services respond with a `304 Not Modified` response. Based on this, the proxy will refresh the response, so that it can continue serving the cached response.
+# The response has not changed. Therefore the backend services respond with a 304 Not Modified response. Based on this, the proxy will refresh the response, so that it can continue serving the cached response.
```

Comment:
```suggestion
# The response has not changed. Therefore, the backend services respond with a 304 Not Modified response. Based on this, the proxy will refresh the response so that it can continue serving the cached response.
```